site_scons/components/__init__.py

```diff
@@ -277,7 +277,15 @@ def define_components(reqs):
     abt_build = ['./configure',
                  '--prefix=$ARGOBOTS_PREFIX',
                  'CC=gcc',
-                 '--enable-stack-unwind']
+                 '--enable-stack-unwind=yes']
+    try:
+        if reqs.get_env('SANITIZERS') != "":
+            # NOTE the address sanitizer library add some extra info on the stack and thus ULTs
+            # need a bigger stack
+            print("Increase argobots default stack size from 16384 to 32768")
+            abt_build += ['--enable-default-stacksize=32768']
```

Comment:
`abt_build += ['--enable-default-stacksize=32768']`
This may not work for these area where custom (bigger than default) stack size is used - see [DSS_DEEP_STACK_SZ](https://github.com/daos-stack/daos/blob/26c2219125f6330c1404461fcc78c0525522ec99/src/include/daos_srv/daos_engine.h#L411).

Reply:
Good point, I will try to find a way to increase It when we are using the libasan.

- [ ] Increase `DSS_DEEP_STACK_SZ` when libasan is used

Reply:
> Good point, I will try to find a way to increase It when we are using the libasan.
> 
> * [x]  Increase `DSS_DEEP_STACK_SZ` when libasan is used

Fixed with commit f38fd44a0